lms/djangoapps/instructor_task/tests/test_tasks_helper.py

```diff
@@ -101,6 +102,7 @@
     'max_size': 2,
     'topics': [{'id': 'topic', 'name': 'Topic', 'description': 'A Topic'}],
 })
+SWITCH_GENERATE_GRADE_REPORT_VERIFIED_ONLY = '.'.join(['instructor_task', GENERATE_GRADE_REPORT_VERIFIED_ONLY])
```

Comment:
Do we have to create switch value like this? Is there any helper method/str method that gives the string representation of the Waffle switch?

Reply:
In other tests, I saw the switch are defined as 
```
@override_switch('course_list_api_rate_limit.rate_limit_2', active=True)
```

I think defining it in the variable is better and reuseable. `SWITCH_GENERATE_GRADE_REPORT_VERIFIED_ONLY`